akmod-specs/displaylink.spec

```diff
@@ -0,0 +1,138 @@
+Name:		displaylink
+Version:	5.4.0
+%global config_release 1
+Release:	1%{?dist}
+Summary:	Meta-package for proprietary DisplayLinkManager application
+URL:		https://www.synaptics.com/products/displaylink-graphics/downloads/ubuntu
+Source0:	https://www.synaptics.com/sites/default/files/exe_files/2021-04/DisplayLink%20USB%20Graphics%20Software%20for%20Ubuntu5.4-EXE.zip
+Source1:	https://github.com/displaylink-rpm/displaylink-rpm/archive/v%{version}.tar.gz
+License:	MIT
+Requires:	akmod-evdi, %{name}-config, %{name}-manager
+
+%description
+Meta-package to install kernel module and proprietary tools
+needed to communicate with and manage DisplayLink USB graphics
+adapters along with additional config files.
+
+%files
+
+%build
+
+%global _hardened_build 1
+
+%package manager
+Summary:	Proprietary DisplayLink Manager application
+License:	Redistributable, no modification permitted
+URL:		https://www.synaptics.com/products/displaylink-graphics/downloads/ubuntu
+ExclusiveArch:	armv7hl i386 x86_64
+BuildRequires:	libdrm-devel
+# The DisplayLinkManager binary is linked at run-time to a specific version of libevdi
+Requires:	libevdi == 1.9.1
+
+%description manager
+This contains the proprietary tools needed to communicate with and manage
+DisplayLink USB graphics adapters.
+
+%files manager
+%license LICENSE
+%dir %{_libexecdir}/%{name}
+%{_libexecdir}/%{name}/DisplayLinkManager
+%{_libexecdir}/%{name}/ella-dock-release.spkg
+%{_libexecdir}/%{name}/firefly-monitor-release.spkg
+%{_libexecdir}/%{name}/ridge-dock-release.spkg
+%dir /var/log/%{name}/
+%{_prefix}/lib/systemd/system-sleep/displaylink.sh
+
+%package config
```

Comment:
Also what's the rationale to separate the "config" package from the "manager" package?
It seems that one cannot live without the other anyway.

Reply:
Config is the files originating from and licensed by this Github project, Manager is the files originating from and licensed by Synaptics/DisplayLink. Since we need multiple packages anyway for the akmod build, splitting config from manager means you get one license per package instead of multiple licenses covering different files in the same package.

Reply:
Hm.., interesting. This could mean that if RPMfusion ever picks this, they could more easily put parts into their separate free/non-free repos; although at the moment it would be useless for someone on the free repo to want to install only the free packages (even if DisplayLink Inc argues evdi can be useful on its own, it hasn't happened so far).

Reply:
Fair enough :-) 👍